src/measure_theory/constructions/borel_space.lean

```diff
@@ -1496,6 +1496,27 @@ begin
   exact h's.closure_eq.symm
 end
 
+/-- The set of points for which a measurable sequence of functions converges is measurable. -/
+@[measurability] lemma measurable_set_exists_tendsto_at_top {ι : Type*}
+  [second_countable_topology α] [complete_space α] [nonempty ι] [encodable ι] [semilattice_sup ι]
+  {f : ι → β → α} (hf : ∀ i, measurable (f i)) :
+  measurable_set {x | ∃ c, tendsto (λ n, f n x) at_top (𝓝 c)} :=
```

Comment:
I tried using `exists_seq_tendsto` but I couldn't figure it out. I see that `∃ c, tendsto (λ n, f n x) at_top (𝓝 c)` shows that it has a subsequence which converges `at_top` but I'm not sure how we can get the reverse. Can I have a hint?

Reply:
I've been thinking about it and I don't find an obvious way to reduce it to the `nat` statement indeed, because even if `ι` is encodable (which I don't think we can avoid ?) the quantification on all subsequences would mean a non-countable intersection.

However, you can do it in a more direct way: if you add 
```
lemma has_antitone_basis.map {f : filter α} {m : α → β}
  {s : ℕ → set α} (hf : has_antitone_basis f s) :
  has_antitone_basis (map m f) (λ n, m '' s n) :=
⟨has_basis.map _ hf.to_has_basis, λ i j hij, image_subset _ $ hf.2 hij⟩
```
in `order.filter.bases`, then you can do: 

```
/-- The set of points for which a measurable sequence of functions converges is measurable. -/
@[measurability] lemma measurable_set_exists_tendsto_at_top
  [hγ : opens_measurable_space γ] [encodable ι] {l : filter ι} [hl : l.ne_bot]
  [l.is_countably_generated] {f : ι → β → γ} (hf : ∀ i, measurable (f i)) :
  measurable_set {x | ∃ c, tendsto (λ n, f n x) l (𝓝 c)} :=
begin
  letI := upgrade_polish_space γ,
  rcases l.exists_antitone_basis with ⟨u, hu⟩,
  simp_rw ← cauchy_map_iff_exists_tendsto,
  change measurable_set {x | _ ∧ _},
  have : ∀ x, ((map (λ i, f i x) l) ×ᶠ (map (λ i, f i x) l)).has_antitone_basis
    (λ n, ((λ i, f i x) '' u n) ×ˢ ((λ i, f i x) '' u n)) := λ x, (hu.map).prod (hu.map),
  simp_rw [and_iff_right (hl.map _), filter.has_basis.le_basis_iff (this _).to_has_basis
    metric.uniformity_basis_dist_inv_nat_succ],
  simp_rw set.set_of_forall,
  refine measurable_set.bInter (countable_encodable _) (λ K _, _),
  simp_rw set.set_of_exists,
  refine measurable_set.bUnion (countable_encodable _) (λ N hN, _),
  simp_rw [prod_image_image_eq, image_subset_iff, prod_subset_iff, set.set_of_forall],
  exact measurable_set.bInter (countable_encodable _) (λ i _,
    measurable_set.bInter (countable_encodable _) (λ j _,
    measurable_set_lt (measurable.dist (hf i) (hf j)) measurable_const))
end
```
Note that some parts of it should probably be extracted and moved to the Cauchy filter API

Reply:
Thanks!